src/libraries/Common/src/Interop/Windows/Kernel32/Interop.WIN32_FILE_ATTRIBUTE_DATA.cs

```diff
@@ -23,6 +23,16 @@ internal void PopulateFrom(ref WIN32_FIND_DATA findData)
                 nFileSizeHigh = findData.nFileSizeHigh;
                 nFileSizeLow = findData.nFileSizeLow;
             }
+
+            internal void PopulateFrom(ref BY_HANDLE_FILE_INFORMATION fileInformationData)
```

Comment:
`ref` => `in`?

Reply:
This is `ref` because the WIN32_FIND_DATA overload does also use this.

https://github.com/dotnet/runtime/blob/83e70174e524f0e762fbae87b6222abdafac0e1b/src/libraries/Common/src/Interop/Windows/Kernel32/Interop.WIN32_FILE_ATTRIBUTE_DATA.cs#L17-L25

Reply:
Why does that impact this? That can also be `in`, but even if it couldn't, why does it need to be the same?

Reply:
It's a maintainability thing.  Passing with `ref` enables the callee to change the argument; passing with `in` prevents that.